pkg/challenges/challenges_test.go

```diff
@@ -66,6 +72,111 @@ func Test_isSpiffeIDAllowed(t *testing.T) {
 	}
 }
 
+func TestUri(t *testing.T) {
+	cfg := &config.FulcioConfig{
+		OIDCIssuers: map[string]config.OIDCIssuer{
+			"https://accounts.domain.com": {
+				IssuerURL:     "https://accounts.domain.com",
+				ClientID:      "sigstore",
+				SubjectDomain: "https://domain.com",
+				Type:          config.IssuerTypeURI,
+			},
+		},
+	}
+	ctx := config.With(context.Background(), cfg)
+	subjectURI := "https://domain.com/users/1"
+	issuer := "https://accounts.domain.com"
```

Comment:
Nit: should `issuer` be called `issuerURI` for consistency with `subjectURI`?

Reply:
Changed `subjectURI` to `subject`